resources/pgp-keys-map.list

```diff
@@ -55,3 +55,40 @@ org.slf4j                       = 0x475F3B8E59E6E63AA78067482C7B12F2A511E325
 org.sonatype.plugins            = 0x2BCBDD0F23EA1CAFCC11D4860374CF2E8DD1BDFD
 
 org.testng                      = 0xDCBA03381EF6C89096ACD985AC5EC74981F9CDA6
+
+com.github.spotbugs:spotbugs-maven-plugin:* = 0x3B9CD44BEEDFBDFABA22717BEEA8F6DF3031CD02
+
+com.google.code.findbugs        = 0x7616EB882DAF57A11477AAF559A252FB1199D873
+
+com.google.errorprone:*:(,2.3.3]= 0xE77417AC194160A3FABD04969A259C7EE636C5ED
```

Comment:
please add example dependency - maybe in test2

Reply:
There exists an entry in `pgp-keys-map-test1`, please see `com.google.errorprone:error_prone_annotations:[2.3.4]`